### ISSUE TITLE
Chat - Wrong error message when members try to open workspace link

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#23188]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
Precondition: User A should have workspace in which he/she is admin and user B is member
1. Open the app and login with user A
2. Open settings
3. Open workspaces
4. Open any workspace in which user B is member
5. Copy the URL and send the URL to user B
6. Open the app in another device and login with user B
7. Click on URL sent by user A and observe the error, error mentions that 'If you need to join the workspace, inform workspace owner' even when though user is already member of workspace

## Expected Result:
App should not mention 'If you want to join the workspace' in error if user is already member of workspace

## Actual Result:
App mentions 'If you want to join the workspace' in error even if user is already member of workspace

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.42-18
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

![wrong message in error](https://github.com/Expensify/App/assets/93399543/8a6cbf25-2364-4d35-a6b7-6eb234682962)

https://github.com/Expensify/App/assets/93399543/e37c3cae-0c1a-4cd2-ac38-a964ccc25119

**Expensify/Expensify Issue URL:**
**Issue reported by:** GH handle - @dhanashree-sawant
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1689705088200759

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by joekaufmanexpensify** [comment]
I don't think this is a bug. Discussing [here. ](https://expensify.slack.com/archives/C01SKUP7QR0/p1689859948405129)

**Comment by joekaufmanexpensify** [comment]
Discussed [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1689861213630709), and determined this is not a bug, as user B is shown this "cannot access" error since they're not an admin. Closing!
